packages/commonwealth/shared/utils.ts

```diff
@@ -71,10 +71,10 @@ export const requiresTypeSlug = (type: ProposalType): boolean => {
 };
 
 /* eslint-disable */
-export const getThreadUrl = (proposal, comment?) => {
-  const aId = proposal.chain;
-  const tId = proposal.type_id || proposal.id;
-  const tTitle = proposal.title ? `-${slugify(proposal.title)}` : '';
+export const getThreadUrl = (type, thread, comment?) => {
```

Comment:
Very worried about this interface change, especially since we don't modify how it's being called anywhere else in the codebase.